testdata/p4tc_samples/simple_l3_varbits.p4

```diff
@@ -0,0 +1,159 @@
+/* -*- P4_16 -*- */
```

Comment:
Please add this testcase to the CMakelist in backends/tc folder and generate output files from compiler and place them

Reply:
As per your suggestion, modified `simple_exact_example.p4` instead of creating a new test case so that the diff is easier to see